src/Source/WMSSource.js

```diff
@@ -135,6 +135,10 @@ class WMSSource extends Source {
         return URLBuilder.bbox(extent, this);
     }
 
+    requestToKey(extent) {
```

Comment:
What about moving this as the default `requestToKey` in `Source` ? If not, we should have a mock method in `Source` to tell that the developer needs to implement this method, like the other methods.

Reply:
moved